app/views/application/_footer.haml

```diff
@@ -11,6 +11,9 @@
       %li
         Read our blog
         = link_to "openaustraliafoundation.org.au/blog", "http://www.openaustraliafoundation.org.au/blog"
+      %li
+        Fork this project
```

Comment:
Could also be 'source code' like on morph.io . or "Source code on GitHub" ?

For people who aren't devs who want to see what's 'behind the scenes' “source code” is more likely a familiar term or something they can work out, I think.